### ISSUE TITLE
Please add support IDEA 2022.2 EAP Thanks.

### ISSUE THREAD

**Issue by dashiyou** [xylo/intellij-postfix-templates#214]
Please add support IDEA 2022.2 EAP Thanks.

**Comment by haocheng** [comment]
@xylo Sorry for bothering but any updates on the 2022.2 support? Thank you 🙏

**Comment by AlNotAlbert** [comment]
no longer working on pycharm 2022.2

**Comment by haocheng** [comment]
I have tried to build a new version of this plugin with #215 and it's working with Intellij IDEA 2022.2 on my laptop (I did not test other IDEs though...). Please feel free to use it if you cannot wait for the PR to be merged. 

[intellij-postfix-templates-2.13.0.222.zip](https://github.com/xylo/intellij-postfix-templates/files/9239594/intellij-postfix-templates-2.13.0.222.zip)

**Comment by haocheng** [comment]
The PR #215 is merged, so please update to latest version from Marketplace instead of using the file above 🙏

**Comment by xylo** [comment]
I'm sorry, that it took so long for me to fix this.  I struggled quite long with settings dialog and finally gave up since I could not find out what needed to be changed to make the plugin working with IDEA 2022.2.  Looks like the line

```
    id "org.jetbrains.intellij" version "1.7.0"
```

did the trick in the end.  So thanks a lot, @haocheng, for your pull request.  I could not merge the code since it's linked to version 2022.1 of this repository, but I applied the changes to the 2022.2 branch and uploaded the new plugin version. :)
